authentik/stages/prompt/tests.py

```diff
@@ -22,6 +22,7 @@
 )
 
 
+# pylint: disable=too-many-public-methods
```

Comment:
should probably break this up into multiple files in the future, but this is fine for now